devbox.json

```
@@ -0,0 +1,26 @@
+{
+  "packages": {
+    "fluxcd": "latest",
+    "awscli2": "latest",
+    "github-cli": "latest",
+    "kustomize": "latest",
+    "pre-commit": "latest",
+    "golangci-lint": "latest",
+    "direnv": "latest",
+    "kubernetes-helm": "latest",
+    "yq-go": "latest",
+    "go": "1.20",
+    "docker": "latest",
+    "gitlint": "latest"
```

Let's add gojq and envsubst in here
```suggestion
    "gitlint": "latest",
    "gojq": "latest",
    "envsubst": "latest"
```